DEVELOPMENT.adoc

```diff
@@ -50,8 +50,9 @@ To debug the plugin, go to the _runIde_ task in the Gradle window and debug it.
 
 Before a release, run Code->Inspect Code->Whole Project (at least) for pkl-hub project and check the results.
 
-The release CI build publishes the plugin zip but doesn't update the plugin update site.
-To update the plugin update site, edit `updatePlugins.xml` on gh-pages branch.
+The release CI build publishes the plugin zip and the `updatePlugins.xml`. +
+The plugin update site always points to the latest released `updatePlugins.xml`, so nothing
+has to be done there.
```

Comment:
The negative instruction ("don't do anything") only makes sense in the context of a previous positive instruction. Maybe leave out altogether, or
```suggestion
The plugin update site is automatically updated.
```

Reply:
Makes sense.